src/settings/Settings.tsx

```diff
@@ -1006,18 +1003,6 @@ export const SETTINGS: { [setting: string]: ISetting } = {
         default: true,
         controller: new UIFeatureController(UIFeature.Voip),
     },
-    "e2ee.manuallyVerifyAllSessions": {
-        supportedLevels: LEVELS_DEVICE_ONLY_SETTINGS,
-        displayName: _td("settings|security|manually_verify_all_sessions"),
-        default: false,
-        controller: new OrderedMultiController([
-            // Apply the feature controller first to ensure that the setting doesn't
-            // show up and can't be toggled. PushToMatrixClientController doesn't
-            // do any overrides anyways.
-            new UIFeatureController(UIFeature.AdvancedEncryption),
-            new PushToMatrixClientController(MatrixClient.prototype.setCryptoTrustCrossSignedDevices, true),
```

Comment:
One could also remove `OrderedMultiController` and `PushToMatrixClientController` since this is the only place that uses them, I suppose.